src/react/account/__tests__/AccountDetails.test.jsx

```diff
@@ -0,0 +1,68 @@
+import * as T from '../../ui-elements/TableKeyValue';
+import AccountDetails from '../AccountDetails';
+import { CustomTabs } from '../../ui-elements/Tabs';
+import React from 'react';
+import { formatDate } from '../../utils';
+import { reduxMount } from '../../utils/test';
```

Comment:
similarly with the ordering; won't write it to all other files but just applies to other relevant files as well :)